services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/document/EnhancedDocument.java

```diff
@@ -464,6 +511,15 @@ interface Builder {
          */
         Builder putJson(String attributeName, String json);
 
+
+        /**
+         * Removes a previously appended attribute.
+         * This can be used where a previously added attribute to the Builder is no longer needed.
+         * @param attributeName The attribute that needs to be removed.
+         * @return Builder instance to construct a {@link EnhancedDocument}
+         */
+        Builder remove(String attributeName);
```

Comment:
What's the use case for this? Do we support this in v1?

Reply:
Its there in V1 as `removeAttribute`, changed this name to removeAttribute now

By Providing remove we can do following   oldDocument.toBuilder().removeAttribute("attr1").build() this will remove attr1 from the